ksql-rest-app/src/main/java/io/confluent/ksql/rest/client/KsqlRestClient.java

```diff
@@ -21,6 +21,19 @@
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import io.confluent.ksql.GenericRow;
```

Comment:
I don't think this re-ordering of imports is not inline with the Google code style we use...

Reply:
IntelliJ did this. I'll check my setting to make sure it's using the correct style file.

Reply:
Again applied our standard style and it kept the import order the same.